luigi/worker.py

```diff
@@ -315,6 +315,18 @@ def __init__(self, scheduler=None, worker_id=None, worker_processes=1, assistant
         self._task_result_queue = multiprocessing.Queue()
         self._running_tasks = {}
 
+        # Stuff for execution_summary
+        self._add_task_history = []
```

Comment:
@ulzha I'm not sure why you keep mentioning to use `_scheduled_tasks`. It's just a dict like `{ "MyTask(a=1)" => MyTask(a=1)}`. It just maps taskids to task objects. It has no information of statuses or ordering.